float64[] rates
int32[] neuron_ids
---
bool success